### PR TITLE
changed: compile each flow simulator as a object library

### PR DESCRIPTION
**Pull request by akva2**
increases parallelism

@atgeirr check if this works better for you.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -272,7 +267,7 @@ opm_add_test(flow_blackoil
   LIBRARIES opmsimulators
   SOURCES
   flow/flow.cpp
-  flow/flow_ebos_blackoil.cpp
+  $<TARGET_OBJECTS:flow_libblackoil>
```
Comment (atgeirr) [inline CMakeLists.txt:270]: Was not this target removed?

**Review by atgeirr — COMMENTED** [review]
My compile time was reduced, although not quite as much as I hoped. I still think the change has merit on its own, as it simplifies the cmake code.

I have a question regarding the flow_libblackoil target.

File: CMakeLists.txt (hunk 1 above)
Comment (akva2) [reply CMakeLists.txt:270]: Nope. It's added in the loop (now there's one such target for each flow simulator).

Comment (atgeirr) [reply CMakeLists.txt:270]: Thanks, now I got it!

**Pull request merged**